.babelrc

```diff
@@ -1,22 +1,12 @@
 {
-  "optional": ["utility.inlineEnvironmentVariables"],
-  "externalHelpers": true,
-  "whitelist": [
-    "es3.memberExpressionLiterals",
-    "es3.propertyLiterals",
-    "es5.properties.mutators",
-    "es6.arrowFunctions",
-    "es6.blockScoping",
-    "es6.classes",
-    "es6.constants",
-    "es6.destructuring",
-    "es6.tailCall",
-    "es6.modules",
-    "es6.parameters",
-    "es6.properties.computed",
-    "es6.properties.shorthand",
-    "es6.templateLiterals",
-    "es6.spread",
-    "utility.inlineEnvironmentVariables"
-  ]
+  "presets": [
+    ["env", {
+      "loose": true,
+      "targets": {
+        "browsers": ["last 2 versions", "safari >= 7"]
```

Comment:
@jridgewell im guessing you want this dynamic based on the original issue, depending on if we're in test or dev mode etc.